src/main/java/ProjectOcean/Controller/CourseController.java

```diff
@@ -17,23 +18,29 @@
 /**
  * Represents the visual component of a course
  */
-public class CourseListIconController extends VBox implements Movable {
+
+public class CourseController extends VBox implements Movable {
+
 
     @FXML private Text courseCodeText;
     @FXML private Text courseNameText;
     @FXML private Text studyPointsText;
     @FXML private VBox typeIndicator;
 
+    private Node owner;
+    private ClipboardContent content;
     private static CoursePlanningSystem model;
```

Comment:
It's not changed here but i'm just curios why model is static? Is it any thought about the decision? Right now a variable is assigned to it at runtime anyway.

Reply:
I would say just to make sure that all courseControllers are using the same model and not potentially different ones.

Reply:
Although the model in CPS is a Singleton.